tests/lib/prepare-project.sh

```diff
@@ -68,17 +68,18 @@ fi
 
 # declare the "quiet" wrapper
 . "$TESTSLIB/quiet.sh"
+. "$TESTSLIB/dirs.sh"
```

Comment:
We could but that seems to clutter spread.yaml a lot more than needed. Actually it would be quite nice if you could tell spread with the environment: list to include necessary scripts by default. something like:

```
environment-initializers:
  - $TESTSLIB/dirs.sh
```